common/script_values/wc_men_at_arms_values.txt

```diff
@@ -0,0 +1,139 @@
+@heavy_cavalry_low_maint_cost = 0.7
+@heavy_cavalry_high_maint_cost = @[heavy_cavalry_low_maint_cost * high_maint_mult]
+
+### Base Cost Values
+@massive_infantry_recruitment_cost = 150
+@massive_infantry_low_maint_cost = @[massive_infantry_recruitment_cost * recr_to_maint_factor]
+@massive_infantry_high_maint_cost = @[massive_infantry_low_maint_cost * high_maint_mult]
+
+@caster_recruitment_cost = 100
+@caster_low_maint_cost = @[caster_recruitment_cost * recr_to_maint_factor]
+@caster_high_maint_cost = @[caster_low_maint_cost * high_maint_mult]
+
+@light_aerial_recruitment_cost = 100
+@light_aerial_low_maint_cost = @[light_aerial_recruitment_cost * recr_to_maint_factor]
+@light_aerial_high_maint_cost = @[light_aerial_low_maint_cost * high_maint_mult]
+
+@massive_aerial_recruitment_cost = 300
+@massive_aerial_low_maint_cost = @[massive_aerial_recruitment_cost * recr_to_maint_factor]
+@massive_aerial_high_maint_cost = @[massive_aerial_low_maint_cost * high_maint_mult]
+
+### Base MAA
+massive_infantry_recruitment_cost = @[massive_infantry_recruitment_cost]
+massive_infantry_low_maint_cost = @[massive_infantry_low_maint_cost]
+massive_infantry_high_maint_cost = @[massive_infantry_high_maint_cost]
+
+caster_recruitment_cost = @[caster_recruitment_cost]
+caster_low_maint_cost = @[caster_low_maint_cost]
+caster_high_maint_cost = @[caster_high_maint_cost]
+
+light_aerial_recruitment_cost = @[light_aerial_recruitment_cost]
+light_aerial_low_maint_cost = @[light_aerial_low_maint_cost]
+light_aerial_high_maint_cost = @[light_aerial_high_maint_cost]
+
+massive_aerial_recruitment_cost = @[massive_aerial_recruitment_cost]
+massive_aerial_low_maint_cost = @[massive_aerial_low_maint_cost]
+massive_aerial_high_maint_cost = @[massive_aerial_high_maint_cost]
+
+### Cultural MAA
+#Add custom costs for each cultural maa here
+### Regional MAA
+#Add custom costs for each regional maa here
+
+##############
+# AI Weights #
+##############
+#Which cultures like what type of unit
+# Warcraft
+culture_ai_weight_caster = {
+	value = 125
+	
+	if = { # Cultures that favours casters
+		limit = {
+			OR = {
+				culture_group = culture_group:elf_group
+				culture = culture:dalaranian
+			}
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike casters
+		limit = {
+			taur_culture_trigger = yes
+			winged_culture_trigger = yes
+		}
+		add = -60
+	}
+}
+culture_ai_weight_massive_infantry = {
+	value = 5
+	
+	if = { # Cultures that favours giants
+		limit = {
+			has_giant_maa_trigger = yes
+		}
+		add = 40
+	}
+}
+culture_ai_weight_light_aerial = {
+	value = 36
+	
+	if = { # Cultures that REALLY like light aerial
+		limit = {
+			winged_culture_trigger = yes
+		}
+		add = 80
+	}
+	if = { # Cultures that favours light aerial
+		limit = {
+			like_aerial_trigger = yes
+		}
+		add = 40
+	}
+	if = { # Cultures that dislike light aerial
+		limit = {
+			dislike_aerial_trigger = yes
+		}
+		add = -60
+	}
+}
+culture_ai_weight_massive_aerial = {
+	value = 120
```

Comment:
Why it's percent on the doc?
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107890754-00f57180-6f34-11eb-8e82-4f7d9d8d5c24.png)

</details>